physics/module_MYNNPBL_wrapper.F90

```diff
@@ -541,6 +543,45 @@ SUBROUTINE mynnedmf_wrapper_run(        &
 !        wspd(i)=wind(i)
       enddo
 
+      ! BWG: Coupling insertion
+      if (cplflx) then
+        do i=1,im
+          if (oceanfrac(i) > zero) then ! Ocean only, NO LAKES
+            if (fice(i) > one - epsln) then ! no open water, use results from CICE
```

Comment:
How about changing if (fice(i) > one -epsln)  to if (.not. wet(i)), so it won't depend on epsln?

Reply:
@benwgreen please have a look. Thanks.

Reply:
Sorry for late response. I am fine with Shan's suggested change.